dp-collection-api.nomad

```diff
@@ -0,0 +1,140 @@
+job "dp-collection-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-collection-api-web" {
```

Comment:
Is there a need for the collection API to be run in web as well as publishing?

Reply:
But I think we did decide to start just with `publishing` as we can change it later. So I'll go ahead and remove this